### DISCUSSION TITLE
Ability to edit your vote in a poll?

### DISCUSSION THREAD

**Discussion by siemensikkema** [community/community#16034] [Discussions]
After casting a vote in a poll it doesn't appear to be possible to change your answer. I think this would be a valuable feature to have if, after some careful convincing, people change their mind.

**Comment by ssbarnea** [comment]
I am perplexed by the fact that once you voted you can no longer change your vote. People can miss-click and one vote can make a huge difference on small surveys... like 99% of them.

**Comment by LindseyB** [comment]
Hi, you should be able to change you vote in a poll. After you vote if you click on `hide results` you are then shown the radio buttons again and then you can change your vote and click on vote again to move your vote to a different answer.
